lib/groups/AdvancedGroup.js

```diff
@@ -44,33 +44,34 @@ class AdvancedGroup extends GroupHelper {
     // }
     resolveOptions() {
         const { args } = this;
+        const options = this.opts.options;
```

Comment:
```suggestion
        const { options } = this.opts;
```

Reply:
At this point you should suggest:

```js
const { args, opts: { options } } = this;
```

:)

Reply:
Deep nesting might affect code readability.